doc/source/user/basics.broadcasting.rst

```diff
@@ -1,22 +1,15 @@
 .. _basics.broadcasting:
+.. _array-broadcasting-in-numpy:
```

Comment:
As a temporary fix, maybe? I'm not entirely sure, though. Maybe @rossbar can help us here

Reply:
I don't think there's anything wrong with having multiple reference names point to the same target (at least, they both seem to work in this case).

Internally, the only document that uses the `array-broadcasting-in-numpy` reference is tutorial-svd, but of course there may be other libraries referencing the target via intersphinx. The approach you've chosen seems sensible to me :+1: